.travis.yml

```diff
@@ -22,6 +23,7 @@ script:
 - make crossbuild
 after_script:
 - goveralls -coverprofile=.profile.cov
+- make deb-systemd
```

Comment:
Oops, This change should be reverted. :bow: 
(I wanted to check `make deb-systemd` works in Travis.)